src/test/java/uk/gov/justice/hmpps/casenotes/services/ExternalApiServiceTest.kt

```diff
@@ -268,13 +267,13 @@ class ExternalApiServiceTest {
         check {
           val components = UriComponentsBuilder.fromUriString(it).build()
           assertThat(components.path).isEqualTo("/api/offenders/{offenderIdentifier}/case-notes/v2")
-          assertThat(components.queryParams).containsExactlyInAnyOrderEntriesOf(
-            mapOf(
-              "size" to listOf("20"),
-              "page" to listOf("5"),
-              "sort" to listOf("creationDateTime,ASCsort=occurrenceDateTime,ASC"),
```

Comment:
comparing a list within the map didn't work as they were different objects, so have had to do it manually